publish.sbt

```diff
@@ -44,26 +45,14 @@ ThisBuild / developers := List(
     url   = url("https://github.com/dk1844")
   ),
   Developer(
-    id    = "AdrianOlosutean",
-    name  = "Adrian Olosutean",
-    email = "adrian.olosutean@absa.africa",
-    url   = url("https://github.com/AdrianOlosutean")
+    id    = "lsulak",
+    name  = "Ladislav Sulak",
+    email = "ladislav.sulak@absa.africa",
+    url   = url("https://github.com/lsulak")
```

Comment:
I was not sure what the process is around these things since his Absa email should be deactivated by now - but I absolutely agree, I'll redo this change.

Reply:
What about the `CODEOWNERS` file, should it be 1:1 to this list of developers, or should it only contain active maintainers?